bower.json

```diff
@@ -28,23 +28,45 @@
     "wct.conf.js"
   ],
   "dependencies": {
-    "iron-icon": "PolymerElements/iron-icon#^1.0.0",
-    "iron-iconset-svg": "polymerelements/iron-iconset-svg#^1.0.0",
-    "paper-button": "PolymerElements/paper-button#^1.0.0",
-    "paper-icon-button": "PolymerElements/paper-icon-button#^1.0.0",
-    "paper-progress": "PolymerElements/paper-progress#^1.0.0",
-    "paper-ripple": "PolymerElements/paper-ripple#~1.0.0",
-    "paper-styles": "PolymerElements/paper-styles#^1.0.0",
-    "polymer": "Polymer/polymer#^1.7.0"
+    "iron-icon": "^1.0.0",
```

Comment:
Is there a reason you guys switched away from specifying the GitHub repo like how the Polymer team does?